ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/SoleTraderDetailsMapper.java

```diff
@@ -56,6 +56,7 @@ public SoleTraderDetails from(CCDCollectionElement<CCDRespondent> ccdSoleTrader)
             .representative(representativeMapper.from(respondent))
             .title(claimantProvidedPartyDetails.getTitle())
             .businessName(claimantProvidedPartyDetails.getBusinessName())
+            .serviceAddress(addressMapper.from(claimantProvidedPartyDetails.getCorrespondenceAddress()))
```

Comment:
What about here...another potential bug you spotted and fixed on the fly?

Reply:
As discussed: we could have pulled this into a separate bug, but since it seems to be redundant we'll just leave the "fix" in place.

Reply:
Thanks for confirming @andrewwa-kainos. Happy with the approach. 👍